glue/flagging_script_glue/flagging.py

```diff
@@ -889,8 +900,19 @@ def outlier_flag(df: pd.DataFrame) -> pd.DataFrame:
         df (pd.DataFrame): dataframe with 'is_outlier' column
     """
 
+    options = ["High price", "Low price", "High price (sqft)", "Low price (sqft)"]
+    pattern = (
+        r"\b(?:" + "|".join(map(re.escape, options)) + r")\b"
+    )  # Creates a pattern with word boundaries
+
     df["sv_is_outlier"] = np.select(
-        [(df["sv_outlier_type"] == "Not outlier")], [0], default=1
+        [
+            df["sv_outlier_reason1"].str.contains(
+                pattern, case=False, na=False, regex=True
+            )
+        ],
+        [1],
+        default=0,
```

Comment:
This code is where we actually decide whether to label an observation as an outlier based on the new `sv_outlier_reason$n` columns.  

For an observation to be flagged as an outlier it needs at least to be a price outlier. In `outlier_type()` the logic puts the price outliers by default in the first column `sv_outlier_reason1`. If there is a price flag - it will be found in this column. So to check if it is an outlier we match on that column with any of the 4 potential price outlier options.